.NET/Microsoft.Recognizers.Text.DateTime/German/Extractors/GermanMergedExtractorConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.German;
+using System.Collections.Generic;
+using Microsoft.Recognizers.Text.Number;
+namespace Microsoft.Recognizers.Text.DateTime.German
+{
+    public class GermanMergedExtractorConfiguration : IMergedExtractorConfiguration
```

Comment:
This is not following the current class design where most configs extend BaseOptionsConfiguration
and in their constructor call the base one to handle options.
Can you please refactor?
Probably this change was merged in after you started creating the German structure.
On further similar merges, I can let you know when they happen.

Reply:
The same applies to other config classes, but not all.
GermanSetExtractorConfiguration, for example, is already on the new model.